src/utils.c

```diff
@@ -181,6 +185,33 @@ bool is_heap(void *addr)
     return p > top_of_kernel_data && p < top_of_ram;
 }
 
+bool is_idevice(void)
+{
+    return chip_id == 0x8960 || chip_id == 0x7000 || chip_id == 0x7001 || chip_id == 0x8000 ||
```

Comment:
This seems like bad naming. There are iDevices with the same chips as Macs. If this is about "old" devices / before a specific cutoff, we should use another function name. If it's about iDevices (e.g. it would hypothetically apply to M1 iDevices) then the test needs to be something else than chip IDs.

Reply:
This is used for both purposes so this function needs to be split.